python/src/MaximumLikelihoodFactory_doc.i.in

```diff
@@ -35,12 +35,32 @@ DistributionFactory
 
 Examples
 --------
+
+In the following example, we estimate the parameters of a 'Normal' distribution 
+with maximum likelihood estimation. 
+
 >>> import openturns as ot
 >>> ot.RandomGenerator.SetSeed(0)
 >>> distribution = ot.Normal(0.9, 1.7)
 >>> sample = distribution.getSample(10)
 >>> factory = ot.MaximumLikelihoodFactory(ot.Normal())
->>> inf_distribution = factory.build(sample)"
+>>> inf_distribution = factory.build(sample)
+
+There are situations where a subset of the parameters 
```

Comment:
Ok. I did not know that the examples in the first section were designed to document the ctor only, but I think that it makes sense.  

So I added a comment at the `setKnownParameter` method. An example was already there, that I did not even notice! With a comment on the purpose of the script, I think that this will be fine.